app/controllers/api/account.php

```diff
@@ -2293,7 +2297,17 @@ function sendSessionAlert(Locale $locale, Document $user, Document $project, Doc
 
         $subject = $locale->getText("emails.otpSession.subject");
         $preview = $locale->getText("emails.otpSession.preview");
+        $heading = $locale->getText("emails.otpSession.heading");
+
         $customTemplate = $project->getAttribute('templates', [])['email.otpSession-' . $locale->default] ?? [];
+        $smtpBaseTemplate = $project->getAttribute('smtpBaseTemplate', 'email-base');
+
+        $validator = new FileName();
+        if (!$validator->isValid($smtpBaseTemplate)) {
+            throw new Exception(Exception::GENERAL_BAD_REQUEST, 'Invalid template path');
```

Comment:
Lets throw server error instead of bad_request, this should be 5xx error. Message still valuable, so we see it on Sentry